app/ts/components/subcomponents/InlineCard.tsx

```diff
@@ -0,0 +1,55 @@
+import { JSX } from 'preact/jsx-runtime'
+import { useSignal, useSignalEffect } from '@preact/signals'
+import { clipboardCopy } from './clipboardcopy.js'
+import { CheckIcon, CopyIcon, EditIcon } from './icons.js'
+
+export type InlineCardProps = {
+	icon: () => JSX.Element
+	label: string
+	value?: string
+	style?: JSX.CSSProperties
+	onEditClick?: JSX.MouseEventHandler<HTMLButtonElement>
```

Comment:
```suggestion
	onEditClicked?: JSX.MouseEventHandler<HTMLButtonElement>
```

Reply:
change included in [43b1a77](https://github.com/DarkFlorist/TheInterceptor/pull/1156/commits/43b1a77124c491f70d7cb85ce1d50385f93e9a3b)